lib/danom/version.rb

```diff
@@ -1,3 +1,3 @@
 module Danom
-  VERSION = "0.1.0"
+  VERSION = "0.1.1"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.

[Read more about it here.](http://www.rubydoc.info/github/bbatsov/rubocop/Rubocop/Cop/Style/StringLiterals)